pkg/resolver/search.go

```diff
@@ -157,38 +162,42 @@ func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool, uid boo
 		ds = goqu.From(schemaTable, jsb)
 	}
 
-	//SELECT CLAUSE
-	if count {
-		selectDs = ds.Select(goqu.COUNT("uid"))
-	} else if uid {
-		selectDs = ds.Select("uid")
-	} else {
-		selectDs = ds.SelectDistinct("uid", "cluster", "data")
-	}
-
-	//WHERE CLAUSE
 	if s.input != nil && (len(s.input.Filters) > 0 || (s.input.Keywords != nil && len(s.input.Keywords) > 0)) {
```

Comment:
We need an `else` to return an error if this validation isn't met.

Reply:
Done.